pkg/notifications/new_discussion_replied_meta.go

```diff
@@ -0,0 +1,23 @@
+package notifications
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/mint-app/pkg/models"
+	"github.com/cmsgov/mint-app/pkg/sqlutils"
+)
+
+// ActivityNewDiscussionRepliedCreate creates an activity for when a Discussion is replied to.
+func ActivityNewDiscussionRepliedCreate(ctx context.Context, np sqlutils.NamedPreparer, actorID uuid.UUID, discussionID uuid.UUID, replyID uuid.UUID, discussionContent models.TaggedHTML, getPreferencesFunc GetUserNotificationPreferencesFunc) (*models.Activity, error) {
+
+	activity := models.NewNewDiscussionRepliedActivity(actorID, discussionID, replyID, discussionContent.RawContent.String())
```

Comment:
I think this should actually be the discussion reply content, not the discussion content.